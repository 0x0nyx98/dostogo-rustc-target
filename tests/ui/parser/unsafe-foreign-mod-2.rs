extern "C" unsafe {
    //~^ ERROR expected `{`, found keyword `unsafe`
    unsafe fn foo();
    //~^ ERROR functions in `extern` blocks cannot have qualifiers
}

fn main() {}
