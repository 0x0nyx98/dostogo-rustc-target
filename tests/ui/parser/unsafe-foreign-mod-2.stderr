error: expected `{`, found keyword `unsafe`
  --> $DIR/unsafe-foreign-mod-2.rs:1:12
   |
LL | extern "C" unsafe {
   |            ^^^^^^ expected `{`

error: functions in `extern` blocks cannot have qualifiers
  --> $DIR/unsafe-foreign-mod-2.rs:3:5
   |
LL | extern "C" unsafe {
   | ----------------- in this `extern` block
LL |
LL |     unsafe fn foo();
   |     ^^^^^^ help: remove this qualifier

error: aborting due to 2 previous errors

